### PR TITLE
Fix `/scripts/issue-genesis-utxo-expenditure.sh`

### PR DESCRIPTION
**Pull request by Jimbo4350**
- Remove `GenesisHash` from config `.yaml` files and specify it on the command line instead.
Relevant: #356

### CODE REVIEW

File: scripts/issue-genesis-utxo-expenditure.sh [hunk 1]
```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env bash
 
-RUNNER=${RUNNER:-cabal new-run -v0 --}
+if [ -e configuration/GenesisFiles/ ]
+then
+    :
+else
+    echo "configuration/GenesisFiles/GENHASH does not exist"
+	/bin/bash ./scripts/genesis.sh
```
Comment (CodiePP) [inline scripts/issue-genesis-utxo-expenditure.sh]: /bin/bash does not exist on every platform, that's why this script is launched via `/usr/bin/env bash`
isn't the `script/genesis.sh` executable?

File: scripts/lib-node.sh [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+then
+    :
+else
+    echo "configuration/GenesisFiles/GENHASH does not exist"
+    /bin/bash ./scripts/genesis.sh
+fi
 
-genesis="$(echo ${genesis_hash} | cut -c-5 | xargs echo -n)"
-genesis_root="configuration/${genesis}"
+genesis_hash="$(echo $(<configuration/GenesisFiles/GENHASH))"
```
Comment (CodiePP) [inline scripts/lib-node.sh]: why not `cat` ?

File: scripts/shelley-testnet2.sh [hunk 3]
```diff
@@ -15,6 +15,7 @@ EXTRA=""
 
 . $(dirname $0)/lib-node.sh
 
+
```
Comment (CodiePP) [inline scripts/shelley-testnet2.sh]: noise.

File: scripts/get-default-key-address.sh [hunk 4]
```diff
@@ -1,10 +1,21 @@
 #!/bin/sh
 
+if [ -e configuration/GenesisFiles/GENHASH ]
+then
+    :
+else
+    echo "configuration/GenesisFiles/GENHASH does not exist"
+	/bin/bash ./scripts/genesis.sh
```
Comment (CodiePP) [inline scripts/get-default-key-address.sh]: /bin/bash does not exist on every platform, that's why this script is launched via /usr/bin/env bash
isn't the script/genesis.sh executable?

File: scripts/lib-node.sh (hunk 2 above)
Comment (Jimbo4350) [reply scripts/lib-node.sh]: No particular reason, I can change it.

**Pull request merged**